mitmproxy/contentviews/image/image_parser.py

```diff
@@ -0,0 +1,34 @@
+from kaitaistruct import KaitaiStream
+
+from . import png
+
+def get_png(data):
+    img = png.Png(KaitaiStream(data))
+    parts = {'format': 'Portable network graphics'}
```

Comment:
Making this a dict is intuitively right, but we need to consider that some keys could occur twice in the metadata. I'd propose to make it a list of `(key, value)` tuples.